examples/your_package_name/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "your_package_name"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[bin]]
+name="simple_publisher"
+path="src/main.rs"
```

Comment:
Given that the counterpart is named `simple_subscriber.rs`, it's better to keep it consistent can name this file `src/simple_publisher.rs`

Reply:
done